README.md

```diff
@@ -1,26 +1,97 @@
+<p align="center"> <img src="https://www.nucleon.sh/img/logo.png"> </p>
 
-This repository contains SumoLogic apps built and maintained by our partners and open source community.
+# Nucleon Cyber app for Sumo Logic
 
-# Contributing
+- [Introduction](#Introduction)
+- [Installation](#Installation)
+- [Dashboard](#Dashboard)
+- [LogMessage](#LogMessage)
+- [Screenshots](#Screenshots)
+- [Support](#Support)
 
-  - To add a new app, reach out to us [here](https://www.sumologic.com/partner-contact/).
+# Introduction
 
-  - To improve the existing app or reporting issues, follow instructions in [CONTRIBUTING](CONTRIBUTING.md)
+Nucleon learns about the external threat landscape in order to determine how threats can
+impact your security posture. Nucleon delivers notifications for pre-attack adversary behaviour, empowering you to truly protect your network. As part of its core functions, the Nucleon polymorphic sensors identify impending threats before impact, automating these early-warnings effectively.  
 
+With our active threat feeds and robust client integrations, these all equal early threat
+detection and visibility, and increased protection. An additional set of these polymorphic sensors sit closer to the gateways of your network, collecting data about sensitive and vulnerable access points that are unique to your network specifically.  
+They expand your attack surface in order to lure in and potentially trap those
+bad actors.  
 
-# Community
-    
-   * For discussions with other Sumo Logic users join our [community forum](https://support.sumologic.com/hc/en-us/community/topics)
-   
-   * You can also join our slack community at sumodojo.slack.com.
-   
-   * Here's the [CODE_OF_CONDUCT](CODE_OF_CONDUCT.md) guidelines to follow.
+We provide only actionable alerts, actively guiding you to block the attack source with the help of your existing defense systems. From the Nucleon dashboard, you can access threat intelligence assessments, threat alerts, insights and remediation guidance that contribute to a strengthened security posture.
 
-# Documentation
+The process is described in detail. For any questions or concerns please contact us through
+your Slack channel or at Support@nucleon.sh
 
-   * To know more about existing apps and integrations follow docs [here](https://help.sumologic.com/07Sumo-Logic-Apps).
+# Installation
+1. Go to SumoLogic home page, and press on ***Setup Wizard***.
+2. Move the cursor to ***Start streaming data to Sumo Logic*** and press on ***Get Started***
+3. Select the following : ***All other Sources***
+4. go to  ***HTTPS Source***
+5. Chose a source category and hit next
+6. copy the URL and send it to Nucleon Cyber (support@cnucleon.sh)
 
+# Dashboard
```

Comment:
Instead of Dashboards it should be Install the Cybereason App and View the Dashboards

Reply:
After the title we can have below lines

```
This page has instructions for installing the Sumo Logic App for Nucleon and descriptions of each of the app dashboards.

Use the instruction from this doc (https://help.sumologic.com/05Search/Library/Apps-in-Sumo-Logic/Install-Apps-from-the-Library) to install the Nucleon App.
```

And then we can continue with dashboard name and other description.